features/bind.feature

```diff
@@ -18,14 +17,13 @@ Feature: Binding
     """
     Then the HTTP response status code is "201"
     And the JSON at "credentials/account" should be "cucumber"
-    And the JSON at "credentials/appliance_url" should be "http://conjur_5"
+    And the JSON at "credentials/appliance_url" should be the master address
     And the JSON at "credentials/authn_login" should be a string
     And the JSON at "credentials/authn_api_key" should be a string
     And the JSON at "credentials/version" should be a Fixnum
     And the JSON at "credentials/ssl_certificate" should be a string
     And the JSON has valid conjur credentials
 
-  @conjur-version-5
```

Comment:
I can't add a comment at this line, but can we update the hardcoded `http://conjur_5-follower` in this scenario to be `the follower address` and follow a pattern similar to the master address?

In your opinion, will these changes ensure the optional follower functionality works for v4 and v5?

Reply:
We could do that, but I'm not sure it would actually be an improvement to the tests. The test in this case is verifying that a service broker that is configured with `CONJUR_FOLLOWER_URL` returns that in the host login configuration instead of `CONJUR_APPLIANCE_URL`.

`CONJUR_APPLIANCE_URL` changes between the tests runs for V4 and V5, so it needs to adapt to that change. However, the value given for `CONJUR_FOLLOWER_URL` is also hard-coded and will be the same in both test runs.

I would be on board with making the URL value version-neutral for the cucumber step. 

For the second question, yes, I believe the follower functionality works as expected for V4 and V5 given these tests.

Reply:
I totally get that in practice these tests are using the same value - I just think for others coming into this later, that might not be obvious. Can we rename the URL to `http://conjur-follower`, if that's easier? I just would like some way to indicate that we're using the same value for both tests, and it is sufficient to prove that the feature works as expected.

Thanks :)

Reply:
@izgeri, that all makes sense!

I made the value for followers a parameter that is passed in for each Conjur version test suite. I also modified the value for the V4 tests to better demonstrate that the tests account for the differences (primarily the `https` protocol and `/api` path, in this case).